src/download/DownloadAsTiff.tsx

```diff
@@ -253,8 +241,23 @@ export function downloadPaintbrushAsTiff(
     });
 
   // Prepare data for export, combining slicesData wi ifds
-  const imageData = getImageData(slicesData, ifds);
+  return getImageData(slicesData, ifds);
+}
 
-  // export image data as tiff file
-  exportImageDataAsTiff(imageData, fileName);
+export function exportImageDataAsTiff(
```

Comment:
My only thought is that the input to this function is not ImageData as the function name suggests... should this not be exportAnnotationsAsTiff?

Reply:
Agreed, will change the name since it no longer takes an `ImageData` object.